Fakebook.Posts/Fakebook.Posts.UnitTests/PostTest.cs

```diff
@@ -22,19 +22,6 @@ public void Post_Constructor_Pass()
             Post newpost = new Post(userEmail, Content);
         }
 
-        [Fact]
-        public void Post_EmailEquality()
-        {
-            //arrange
-            const string Content = "This is some content";
-            const string userEmail = "damion.silver@test.com";
-            //act
-            Post newpost = new Post(userEmail, Content);
-
-            //assert
-            Assert.Equal(userEmail, newpost.UserEmail);
-        }
-
```

Comment:
why did this one (Post_EmailEquality) get removed, this was a good one. pay attention to the comments, this is very sloppy

Reply:
I added this test back to the commit. I deleted more than I intended to delete. I am sorry for it being sloppy.